streaming/base/converters/dataframe_to_mds.py

```diff
@@ -201,12 +159,10 @@ def write_mds(iterator: Iterable):
         else:
             raise RuntimeError('TaskContext.get() returns None')
 
-        if isinstance(mds_path, str):  # local
-            output = os.path.join(mds_path, f'{id}')
-            out_file_path = output
+        if mds_path[1] == '':
```

Comment:
Can you please add a comment when will `mds_path[1]` be empty ?

Reply:
Add. It is for cases when out is local-only.